src/services/panthalassa/index.js

```diff
@@ -0,0 +1,318 @@
+ * @returns {Promise<*>} {string} Account's signed profile
+ */
+export async function panthalassaSignProfileStandAlone(name: String, location: String, image: String, keyManagerStore: String, password: String):Promise<string> {
+  return Panthalassa.PanthalassaSignProfileStandAlone({
+    name,
+    location,
+    image,
+    keyManagerStore,
+    password,
+  });
+}
+
+/**
+ * @desc Updates information to the Account's profile
+ * @param {string} name Profile's name to show
+ * @param {string} location Profile's location
+ * @param {string} image Profile's avatar
+ * @returns {Promise<*>} {
+ */
+export async function panthalassaSignProfile(name: String, location: String, image: String):Promise<string> {
+  return Panthalassa.PanthalassaSignProfile({
+    name,
+    location,
+    image,
+  });
+}
+
+/**
+ * @desc Retrieves Account's mnemonic phrase
+ * @returns {Promise<*>} {string} Mnemonic phrase
+ */
+export async function panthalassaGetMnemonic():Promise<string> {
+  return Panthalassa.PanthalassaGetMnemonic();
+}
+
+/**
+ * @desc Starts DApp's VM
+ * @param {string} dApp DApp's Identifier
+ * @param {string} timeout Value for timeout operation
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaStartDApp(dApp: String, timeout: Number):Promise<boolean> {
+  return Panthalassa.PanthalassaStartDApp({
+    dApp,
+    timeout,
+  });
+}
+
+/**
+ * @desc Opens an specific Dapp
+ * @param {string} id DApp's Id
+ * @param {string} context TODO
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaOpenDApp(id: String, context: String):Promise<boolean> {
+  return Panthalassa.PanthalassaOpenDApp({
+    id,
+    context,
+  });
+}
+
+/**
+ * @desc TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaNewPreKeyBundle():Promise<string> {
+  return Panthalassa.PanthalassaNewPreKeyBundle();
+}
+
+/**
+ * @desc Calls a DApp's function over th VM
+ * @param {string} dAppId TODO
+ * @param {string} id TODO
+ * @param {string} args TODO
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaCallDAppFunction(dAppId: String, id: Number, args: String):Promise<boolean> {
+  return Panthalassa.PanthalassaCallDAppFunction({
+    dAppId,
+    id,
+    args,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} address TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaConnectToDAppDevHost(address: String):Promise<boolean> {
+  return Panthalassa.PanthalassaConnectToDAppDevHost(address);
+}
+
+/**
+ * @desc TODO
+ * @param {string} rawMsg TODO
+ * @param {string} secretID TODO
+ * @param {string} secret TODO
+ * @param {string} receiverIdKey TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaCreateDAppMessage(rawMsg: String, secretID: String, secret: String, receiverIdKey:String):Promise<string> {
```

Comment:
I don't think it's needed, since chat is moved to Panthalassa.

Reply:
Deleted